### ISSUE TITLE
Enregistrer et quitter n'a pas le comportement attendu

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/gn_module_import#170]
Actuellement, je suis à l'étape 2, je choisis un mapping. Si j'enregistre et quitte, je reviens à l'étape 2 mais sans mapping sélectionné. 

Dans le pire des cas, un utilisateur peut se créer un mapping, faire les correspondances, appuyer sur "enregistrer et quitter", il devra recommencer, le mapping n'est pas enregistré.

**Comment by mvergez** [comment]
Effectivement, j'ai l'impression, en regardant le code que "Enregistrer et Quitter" n'a aucun effet : 

Quand on clique sur le bouton, on déclenche `onImportList()` :
https://github.com/PnX-SI/gn_module_import/blob/6e7561ee5de223488d329157c3418ee887368ff1/frontend/app/components/import_process/footer-stepper/footer-stepper.component.html#L11-L18

Et voici le contenu de cette méthode :
https://github.com/PnX-SI/gn_module_import/blob/6e7561ee5de223488d329157c3418ee887368ff1/frontend/app/components/import_process/footer-stepper/footer-stepper.component.ts#L57-L59

Et le contenu de `resetStepoer()` :
https://github.com/PnX-SI/gn_module_import/blob/6e7561ee5de223488d329157c3418ee887368ff1/frontend/app/components/import_process/steps.service.ts#L92-L98

Il faudrait que le footer component renvoie un signal qui est traité dans chaque étape. J'essaierai de faire une PR dans ce sens

**Comment by camillemonchicourt** [comment]
Je pense que le soucis est réglé avec la v2 ?
